spinalcordtoolbox/gui/widgets.py

```diff
@@ -140,20 +147,22 @@ def _init_ui(self, data, aspect):
 
         if self._crosshair:
             self.cursor = Cursor(self._axes, useblit=True, color='r', linewidth=1)
+
         self.points = self._axes.plot([], [], '.r', markersize=7)[0]
 
     def title(self, message):
-        self._fig.suptitle(message)
+        self._fig.suptitle(message, fontsize=10)
 
     def annotate(self, x, y, label):
         self.annotations.append(self._axes.annotate(label, xy=(x, y), xytext=(-3, 3),
                                                     textcoords='offset points', ha='right',
                                                     va='bottom', color='r'))
 
     def clear(self):
-        for i in self.annotations:
-            i.remove()
-        self.annotations = []
+        if hasattr(self, 'annotations'):
```

Comment:
I think it would be better to always have this attribute here. And then you keep it as an empty list when there are no annotations. Then you can do something like:

```
for i in self.annotations:
    i.remove()
self.annotations.clear()
```

without having to do the check using `hasattr()`.